components/sandbox/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ResizableIframe from 'components/resizable-iframe';
+
+const Sandbox = React.createClass( {
+	displayName: 'Sandbox',
+
+	propTypes: {
+		html: React.PropTypes.string,
```

Comment:
`PropTypes` is moved out of React as of version 16. I'd expect this is logging a warning in the console. We should probably just drop `propTypes` here altogether.